docs/pages/management/register-services/register-services-join-token.mdx

```diff
@@ -0,0 +1,253 @@
+</Notice>
+
+Static tokens are defined ahead of time by an administrator and stored in the
+auth server's config file:
+
+```yaml
+# Config section in `/etc/teleport.yaml` file for the auth server
+auth_service:
+    enabled: true
+    tokens:
+    # This static token allows new hosts to join the cluster as "proxy" or "node"
+    - "proxy,node:secret-token-value"
+    # A token can also be stored in a file. In this example the token for adding
+    # new auth servers are stored in /path/to/tokenfile
+    - "auth:/path/to/tokenfile"
+```
+</Details>
+
+### Start your service with the invite token
+
+Execute the following commands on the host where you will run your service so you
+can use the `INVITE_TOKEN` variable to start Teleport.
+
+```code
+$ export INVITE_TOKEN=<Var name="invite-token" />
+```
+
+Still on the host where you will run your service, assign the address of your
+Teleport Cloud tenant to an environment variable, including the domain name (or
+IP address) and port `443`:
+
+```code
+$ export ADDR=mytenant.teleport.sh:443
+```
+
+<Details title="Ensuring the correct ports are opened">
+
+If you are have disabled TLS Routing, the Teleport Proxy Service listens on
+separate ports in order to register new services, rather than performing all
+activity via port 443. 
+
+Using the ports in Teleport's default configuration, your service needs to be
+able to talk to ports `3080` and `3024` on the Proxy Service. Port `3080` is
+used to initially fetch the credentials (SSH and TLS certificates) and for
+discovering the reverse tunnel. Port `3024 `is used to establish a connection to
+the Auth Service through the Proxy.
+
+</Details> 
+
+<Details
+  type="tip"
+  title="Using multiple Proxy Service instnaces behind a load balancer"
+>
+
+  The setup above also works even if the cluster uses multiple Proxy Service
+  instances behind a load balancer (LB) or a DNS entry with multiple values. In
+  this case, the service establishes a tunnel to every Proxy Service instance.
+  
+  This requires that an LB use a round-robin or a similar balancing algorithm.
+  Do not use sticky load balancing algorithms (a.k.a. "session affinity") with
+  Teleport Proxy Service instances.  
+
+</Details>
+
+Execute the following command on your service to add it to a cluster:
+
+```code
+$ sudo teleport start \
+   --roles=node \
+   --token=${INVITE_TOKEN?} \
+   --auth-server=${ADDR?}
+```
+
+As new services come online, they start sending ping requests every few seconds to
+the Auth Service. This allows users to explore cluster membership and size.
+
+Run the following command on your local machine to see all of the Teleport services
+in your cluster:
+
+```code
+$ tctl nodes ls
+
+service Name     service ID                                  Address            Labels
+---------     -------                                  -------            ------
+turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro:ubuntu
```

Comment:
You'll only see port 3022 like this when services register via the auth server, but we've now standardized on registering via proxy, so this column would say `tunnel` instead.